src/components/NodeUserInput.tsx

```diff
@@ -81,26 +85,19 @@ export const NodeUserInput = (props) => {
         return
       }
 
-      const inputs = {
-        claimId: { id: 'entry.1356714542', value: nodeId },
-        argumentId: { id: 'entry.1531525675', value: argumentId },
-        forOrAgainst: { id: 'entry.394432412', value: forOrAgainst },
-        claim: { id: 'entry.2014411862', value: claim },
-        timestamp: { id: 'entry.1370532301', value: timestamp },
-      }
-
-      let requestUrl = formConfig.formUrl
-
-      Object.keys(inputs).forEach((key) => {
-        requestUrl += `${inputs[key].id}=${inputs[key].value}&`
-      })
-
-      requestUrl += 'submit=Submit'
-
-      await axios({
-        url: encodeURI(requestUrl),
+      await axios(encodeURI(formConstants.url), {
         method: 'get',
-        contentType: 'application/x-www-form-urlencoded',
+        headers: {
```

Comment:
I suspect setting the headers this way (rather than adding this to the headers) is causing the issue where the form submissions don't get logged in the responses.

Reply:
It turns out axios does want the headers in an object but the keys need to be formatted correctly, like 'Content-Type'